cli/deal/inspect.go

```diff
@@ -0,0 +1,80 @@
+package deal
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/fatih/color"
+	lapi "github.com/filecoin-project/lotus/api"
+)
+
+func InspectDealCmd(ctx context.Context, api lapi.FullNode, proposalCid string, dealId int) error {
```

Comment:
Why a separate function in a separate package? All other commands are just implemented in the cli package.

Reply:
Just legacy from the initial UI we were building. I moved to the `cli` package!